### PR TITLE
Removed the "cc-catalog" references from the system-architecure diagram. 

### LINKED ISSUE #758: Remove CC-Catalog references in API docs diagram in readme
## Description

> The API readme has references to CC-Catalog in the System Architecture diagram

Reported by @obulat 

We should re-create the diagram for Openverse. Use a tool that can export the diagram to XML or JSON or something so we can add it to the repository and easily update it again in the future.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by raghuvar-arora-au2**
## Fixes
Fixes #758  by @sarayourfriend 

## Description
Recreated the diagram in draw.io, and removed the cc-catolog references. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by raghuvar-arora-au2** [comment]
Where should I put the XML file and the link to the draw.io project?

**Comment by zackkrida** [comment]
@raghuvar-arora-au2 how about moving all of this into a subdirectory of the `readme_assets` folder, maybe something like:

```
readme_assets
└── system_diagram
    ├── diagram.png
    ├── diagram.xml
    └── readme.md
```

and then the link to draw.io could go in that readme file? if you do it this way, make sure to update path to the image in the repo README file. 

The diagram looks excellent! Thanks for this.

**Comment by raghuvar-arora-au2** [comment]
Made the changes, thanks

File: readme_assets/system_diagram/system_architecture.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-21" value="Redis" style="strokeWidth=1;html=1;shape=mxgraph.flowchart.database;whiteSpace=wrap;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="520" y="250" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-58" value="-Search queries&lt;br&gt;-Tag Stats&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
+          <mxGeometry x="770" y="350" width="110" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-59" value="-Site cache&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Traffic stats&amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Dead link cache" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
```
Comment (sarayourfriend) [inline readme_assets/system_diagram/system_architecture.xml:88]: We no longer hold traffic stats anywhere as far as I am aware, so I think we can remove that bit.

```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-59" value="-Site cache&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Dead link cache" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="455" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="545" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="635" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
```
Comment (sarayourfriend) [inline readme_assets/system_diagram/system_architecture.xml:58]: I think 'django-api' might be a better term to use here as "catalog" could be easily confused with our https://github.com/wordpress/openverse-catalog project.

```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="455" y="480" width="80" height="40" as="geometry" />
        </mxCell>
        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="545" y="480" width="80" height="40" as="geometry" />
        </mxCell>
        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="635" y="480" width="80" height="40" as="geometry" />
        </mxCell>
```

**Review by sarayourfriend — COMMENTED** [review]
This is great @raghuvar-arora-au2! It looks like you'll need to run `just lint` at the root of the repository and commit the changes to the markdown file, but otherwise this is great to see :slightly_smiling_face: 

I left a couple things that would be nice to update while we're making this change as well.

Additionally, I don't think the `shares.cc` service is running anymore, but maybe @zackkrida could confirm that. We might want to remove that block as well.

Thanks again for the contribution!!

**Review by obulat — CHANGES_REQUESTED** [review]
Thank you for amazing work, @raghuvar-arora-au2 !

There are a couple of things that we don't have anymore, could you please remove them. I've scratched them out in the screenshot below:
Screen Shot 2022-06-27 at 2 34 04 PM

**Comment by stacimc** [comment]
This is looking great! 🎉 

Can we update the text "Ingest" here to say "Data refresh"? Context in [comment](https://github.com/WordPress/openverse-api/issues/758#issuecomment-1167534569). (I do think it's correct to keep the phrase "Ingestion server", since the server is currently named that).

Screen Shot 2022-06-27 at 2 34 04 PM

**Review by dhruvkb — APPROVED** [review]
I'm not sure if we should be including the frontend in the diagram that's in the API repo, but it's not wrong so not a problem.

One technical error in the "API Database" ↔ "API Server Group" link is that we no longer have link shortening data and lists. Those tables exist but that functionality has been deactivated.

Also a couple of grammatical improvements:
- "Django-api" &rarr; "Django API"
- "frontend" &rarr; "Frontend"

File: readme_assets/system_diagram/system_architecture.xml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+          <mxGeometry x="690" y="210" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-25" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0.5;entryY=1;entryDx=0;entryDy=0;entryPerimeter=0;startArrow=classic;startFill=1;endArrow=none;endFill=0;exitX=1;exitY=0.5;exitDx=0;exitDy=0;" parent="1" source="JVIGxF9oKAkimninJbSp-10" target="JVIGxF9oKAkimninJbSp-9" edge="1">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-10" value="API Server Group&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#dae8fc;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="435" y="430" width="300" height="130" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="Django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:50]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="Django API" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+          <mxGeometry x="455" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="Django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:53]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="Django API" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+          <mxGeometry x="545" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="Django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:56]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="Django API" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" parent="1" vertex="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-72" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;labelBackgroundColor=#F8CECC;startArrow=none;startFill=0;endArrow=classic;endFill=1;entryX=1;entryY=0.5;entryDx=0;entryDy=0;" parent="1" source="JVIGxF9oKAkimninJbSp-15" target="JVIGxF9oKAkimninJbSp-71" edge="1">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="381.7391357421875" y="610" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-15" value="Load Balancer" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#dae8fc;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="525" y="590" width="120" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-16" value="frontend" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#f8cecc;strokeColor=#b85450;" parent="1" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:70]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-16" value="Frontend" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#f8cecc;strokeColor=#b85450;" parent="1" vertex="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-63" value="EC2&amp;nbsp; Instances" style="shape=partialRectangle;html=1;whiteSpace=wrap;connectable=0;fillColor=#ffe6cc;top=0;left=0;bottom=0;right=0;overflow=hidden;pointerEvents=1;align=left;spacingLeft=10;strokeColor=#d79b00;" parent="JVIGxF9oKAkimninJbSp-62" vertex="1">
+          <mxGeometry width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-64" value="Other infrastructure" style="shape=partialRectangle;html=1;whiteSpace=wrap;connectable=0;fillColor=#dae8fc;top=0;left=0;bottom=0;right=0;overflow=hidden;pointerEvents=1;align=left;spacingLeft=10;strokeWidth=1;strokeColor=#6c8ebf;" parent="JVIGxF9oKAkimninJbSp-62" vertex="1">
+          <mxGeometry y="30" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-65" value="External dependancy" style="shape=partialRectangle;html=1;whiteSpace=wrap;connectable=0;fillColor=#f8cecc;top=0;left=0;bottom=0;right=0;overflow=hidden;pointerEvents=1;align=left;spacingLeft=10;strokeWidth=1;strokeColor=#b85450;" parent="JVIGxF9oKAkimninJbSp-62" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:106]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-65" value="External dependency" style="shape=partialRectangle;html=1;whiteSpace=wrap;connectable=0;fillColor=#f8cecc;top=0;left=0;bottom=0;right=0;overflow=hidden;pointerEvents=1;align=left;spacingLeft=10;strokeWidth=1;strokeColor=#b85450;" parent="JVIGxF9oKAkimninJbSp-62" vertex="1">
```

File: readme_assets/system_diagram/system_architecture.xml [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+          <mxGeometry x="770" y="350" width="110" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-59" value="-Site cache&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Dead link cache" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" parent="1" vertex="1">
+          <mxGeometry x="590" y="360" width="125" height="41" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-60" value="-Detail views&lt;br&gt;-Lists&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&lt;br&gt;-Content provider data&lt;br&gt;&lt;div style=&quot;&quot;&gt;&lt;span style=&quot;background-color: initial;&quot;&gt;-Link shortening data&lt;/span&gt;&lt;/div&gt;" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" parent="1" vertex="1">
```
Comment (obulat) [inline readme_assets/system_diagram/system_architecture.xml:91]:
```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-60" value="-Detail views&lt;br&gt;-Lists&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&lt;br&gt;-Content provider data&lt;br&gt;&lt;div style=&quot;&quot;&gt;&lt;span style=&quot;background-color: initial;&quot;&gt;&lt;/span&gt;&lt;/div&gt;" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" parent="1" vertex="1">
```

Comment (obulat) [reply readme_assets/system_diagram/system_architecture.xml:91]: I'm not sure that this suggestion will work for PNG generation, but we should remove the `Link shortening data` here, too.

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @raghuvar-arora-au2 ! The diagram looks great 🎉

**Comment by AetherUnbound** [comment]
Just dropping in to say thanks so much for this contribution @raghuvar-arora-au2! This is excellent 🥳

**Pull request merged**